web/src/urls.tsx

```diff
@@ -1,55 +1,56 @@
-import * as React from 'react';
+import * as React from "react";
 
-import DocumentationArticle from './docs/Documentation'
-import SwaggerUI from 'swagger-ui-react'
-import {
-    Routes as Switch,
-    Route,
-    Link
-} from "react-router-dom";
-import { ProjectSummary } from './project/ProjectSummary';
-import ProjectsAdmin from './admin/ProjectsAdmin';
-
-
-const Index = () => {
-
-    return (<div>
-        <h2>Sample metadata server</h2>
-        <ul>
-            <li><Link to="documentation">Python API documentation</Link></li>
-            <li><Link to="swagger">Swagger page</Link></li>
-            <li><Link to="project">Samples table</Link></li>
-        </ul>
-
-        <DocumentationArticle articleId="index" />
-
-    </div>)
-}
+import DocumentationArticle from "./docs/Documentation";
+import SwaggerUI from "swagger-ui-react";
+import { DetailedInfoPage } from "./DetailedInfoPage";
+import { Routes as Switch, Route } from "react-router-dom";
+import { ProjectSummary } from "./project/ProjectSummary";
+import ProjectsAdmin from "./admin/ProjectsAdmin";
 
 export const Routes = () => {
-
-    return <>
-        <Switch>
-            <Route
-                path="/documentation"
-            >
-                <Route path="" element={<DocumentationArticle />} />
-                <Route path=":id" element={<DocumentationArticle />} />
-            </Route>
-
-            <Route
-                path="/swagger"
-                element={<SwaggerUI url="/openapi.json" />}
-            />
-
-            <Route path="project" element={<ProjectSummary />} />
-
-            <Route path="admin" element={<ProjectsAdmin />} />
-
-            <Route
-                path="/"
-                element={<Index />}
-            />
-        </Switch>
-    </>
-}
+    return (
+        <>
+            <Switch>
+                <Route path="/documentation">
+                    <Route path="" element={<DocumentationArticle />} />
+                    <Route path=":id" element={<DocumentationArticle />} />
+                </Route>
+
+                <Route
+                    path="/swagger"
+                    element={<SwaggerUI url="/openapi.json" />}
+                />
+
+                <Route path="project/" element={<ProjectSummary />} />
+                <Route
+                    path="project/:projectName"
+                    element={<ProjectSummary />}
+                />
+                <Route
+                    path="project/:projectName/:page/:pageSize"
```

Comment:
IMO, pageSize should remain a query param

Reply:
Fixed